WalletWasabi/Wallets/WalletManager.cs

```diff
@@ -262,6 +267,8 @@ private void AddWallet(Wallet wallet)
 			wallet.WalletRelevantTransactionProcessed += TransactionProcessor_WalletRelevantTransactionProcessed;
 			wallet.OnDequeue += ChaumianClient_OnDequeue;
 			wallet.StateChanged += Wallet_StateChanged;
+
+			WalletAdded.Invoke(this, wallet);
```

Comment:
Do we need to check null here?

```suggestion
			WalletAdded?.Invoke(this, wallet);
```

Reply:
Such a shame on me...